### DISCUSSION TITLE
Error while compiling the game (bug fixed, but not in the end)

### DISCUSSION THREAD

**Discussion by doggywatty** [KadeArchive/Kade-Engine#597] [Q&A]
Well, when I was compiling the game, but I got an error: C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
source/PlayState.hx:299: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
C:\Users\Usuario\Downloads\Kade-Engine-master\Kade-Engine-master\export\release\windows\obj>setlocal enabledelayedexpansion
Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths
Error: Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths

I solved it, I had to go to PlayState.hx and in line 299, it said FlxCamera.defaultCameras, I had to change to FlxG.cameras.setDefaultDrawTarget, it was solved but it tells me this ANOTHER ERROR:
source/PlayState.hx:299: characters 3-36 : Cannot rebind this method : please use 'dynamic' before method declaration
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access

I solved it, again, in line 299 it said: FlxG.cameras.setDefaultDrawTarget = [cameras];
then where it says cameras, change it to dynamic
it was fixed!
But it gave me another error:
source/PlayState.hx:299: characters 40-47 : Expected expression

I dont know this part. Can someone help me?

**Comment by isakube** [comment]
_aaaaaaaa aaa aa a_

Deprecations don't matter. You should be fine with polymod.

Since the compiler expected an expression, perhaps try changing that line to `FlxG.cameras.setDefaultDrawTarget([cameras]);`

**Comment by doggywatty** [comment]
hey, now its different:
source/PlayState.hx:299: characters 37-45 : Array<flixel.FlxCamera> should
be flixel.FlxCamera
source/PlayState.hx:299: characters 37-45 : For function argument 'camera'
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189:
characters 10-23 : Warning : This typedef is deprecated in favor of
haxe.xml.Access
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189:
characters 30-43 : Warning : This typedef is deprecated in favor of
haxe.xml.Access

**Comment by isakube** [comment]
> hey, now its different: source/PlayState.hx:299: characters 37-45 : Array<flixel.FlxCamera> should be flixel.FlxCamera source/PlayState.hx:299: characters 37-45 : For function argument 'camera' 

remove the square brackets i think

**Comment by doggywatty** [comment]
its the same

source/PlayState.hx:299: characters 37-44 : Array<flixel.FlxCamera> should
be flixel.FlxCamera
source/PlayState.hx:299: characters 37-44 : For function argument 'camera'
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189:
characters 10-23 : Warning : This typedef is deprecated in favor of
haxe.xml.Access
C:/Users/Usuario/haxelib/polymod/git/polymod/format/XMLMerge.hx:189:
characters 30-43 : Warning : This typedef is deprecated in favor of
haxe.xml.Access

**Comment by isakube** [comment]
hmmm... well, i have no idea. try compiling the latest source code and see if that fixes anything. I hate that this is how i contribute to issues, but it's pretty much the only advice i have most of the time, soooo

**Comment by doggywatty** [comment]
i tried the latest source code and nothing. nothing is working
